mercapi/mercapi.py

```diff
@@ -22,19 +24,28 @@ class Mercapi:
 
     **Avoid instantiating this class more than once in a single runtime.**
     """
-
-    _user_agent = (
-        "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
-    )
-    _headers = {
-        "User-Agent": _user_agent,
-        "X-Platform": "web",
-    }
-
-    def __init__(self):
+    
+    def __init__(
+        self,
+        *,
+        proxies: typing.Optional[ProxiesTypes] = None,
```

Comment:
`Optional` is imported directly with `from typing import Optional`. I don't understand why you'd re-import the entire `typing` module to use `typing.Optional` here.